Sources/Extensions/UIKitExtension.swift

```diff
@@ -23,12 +23,12 @@ public extension UITableView {
         ) {
         reload(
             using: stagedChangeset,
-            deleteSectionsAnimation: animation,
-            insertSectionsAnimation: animation,
-            reloadSectionsAnimation: animation,
-            deleteRowsAnimation: animation,
-            insertRowsAnimation: animation,
-            reloadRowsAnimation: animation,
+            deleteSectionsAnimation: animation(),
```

Comment:
Even tho podspec changes say supported Swift 4.2 - this is not supported in Swift 4.2. Actually this PR is more like migration than just updates. 😄

Reply:
That's strange, the project has been set to use Swift 4.2 and compiles just fine here. Or do you mean something else besides compilation is the problem?

Reply:
Those changes are necessary because autoclosure forwarding behaviour has changed.